lib/commanded/event/handler.ex

```diff
@@ -448,7 +547,8 @@ defmodule Commanded.Event.Handler do
     `Commanded.Event.FailureContext` struct as described in `{:retry, context}`
     above.
 
-  - `:skip` - skip the failed event by acknowledging receipt.
+  - `:skip` - skip the failed event by acknowledging receipt. Note that this is not
+    a valid return value in batching mode.
```

Comment:
Is this comment still applicable, as it appears that `:skip` is supported for batch errors?

Reply:
Thanks for catching that, `:skip` is indeed supported, this just got missed as part of that. I've taken that bit out and clarified it's affects on batching